packages/hash/external-services/kratos/identity.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$id": "https://schemas.ory.sh/presets/kratos/quickstart/email-password/identity.schema.json",
```

Comment:
If we've changed this schema, I think we should change the ID here :)

Reply:
Good point, we should have our own `$id` - do you have any thoughts what it should be? Perhaps something like:
```
https://hash.ai/schemas/v1/kratos-identity.schema.json"
```

We should be careful to not confuse this JSON Schema with our HASH Graph Type JSON Schemas

Reply:
Definitely could be confusing with the other JSON Schemas we have. I think that's fine for the $id, though. I assume we won't be making this publicly available, though?

Reply:
It would be publicly available in the repo of course, but it should only be considered as a part of our kratos definition/configuration so not really something we'd point anyone to.

Reply:
Have modified the `$id` in https://github.com/hashintel/hash/pull/889/commits/145e7822ff9a695b4f9cc0f46bb59af832b6bb18